api/external_test.go

```diff
@@ -37,9 +37,109 @@ func TestExternal(t *testing.T) {
 }
 
 func (ts *ExternalTestSuite) SetupTest() {
+	ts.Config.DisableSignup = false
```

Comment:
This file has gotten pretty large, it might help a bit to split tests out by the provider?